adm/_metwork.spec

```diff
@@ -393,15 +410,22 @@ EOF
             if test -d /home/{{MFMODULE_LOWERCASE}}; then
                 echo "INFO: saving old /home/{{MFMODULE_LOWERCASE}} to /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX} ..."
                 mv /home/{{MFMODULE_LOWERCASE}} /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}
+                mkdir /home/{{MFMODULE_LOWERCASE}}
+                #Keep .ssh directory for the next install
+                if test -d /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/.ssh; then
+                    cp -Rp /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/.ssh /home/{{MFMODULE_LOWERCASE}}
```

Comment:
add a / (slash) after `/home/{{MFMODULE_LOWERCASE}}` !? to copy to a sub-dir, not to overwritte existing dir

Reply:
Ok, even if it works without...